programs/biome.nix

```diff
@@ -31,6 +31,12 @@ in
   options.programs.biome = {
     enable = l.mkEnableOption "biome";
     package = l.mkPackageOption p "biome" { };
+    config-path = l.mkOption {
+      description = "The path to the Biome configuration file.";
+      type = t.str;
+      example = "/path/to/config/directory";
+      default = "";
```

Comment:
use [types.path](https://github.com/NixOS/nixpkgs/blob/master/lib/types.nix#L517-L522), and change the example to mention the config _file_ instead of the directory